src/utils/isSubsetOf.ts

```diff
@@ -0,0 +1,39 @@
+/** Taken from TypeScript collection lib to perfectly match the .isSubsetOf signature */
+export interface ReadonlySetLike<T> {
+    /**
+     * Despite its name, returns an iterator of the values in the set-like.
+     */
+    keys(): Iterator<T>;
+    /**
+     * @returns a boolean indicating whether an element with the specified value exists in the set-like or not.
+     */
+    has(value: T): boolean;
+    /**
+     * @returns the number of (unique) elements in the set-like.
+     */
+    readonly size: number;
+}
+
+/**
+ * @returns a boolean indicating whether all the elements in Set `one` are also in the `other`.
+ */
+export function isSubsetOf<T>(
+    one: Set<T>,
+    other: ReadonlySetLike<unknown>,
+): boolean {
+    // If the builtin method exists, just call it
+    /* eslint-disable @typescript-eslint/no-explicit-any */
+    if ((Set.prototype as any).isSubsetOf) {
```

Comment:
```suggestion
    if ('isSubsetOf' in Set.prototype) {
```

Reply:
We'll add eslint rules that disable both `as` and `any` (except for one only file) to ensure types are at least somewhat sound. It's hard to do enable them right now, but it'd be nice to not add any more :)